### PR TITLE
add config option to move the fetch key box above the bookmarks

### PR DESCRIPTION
**Pull request by Olezha**
https://freenet.mantishub.io/view.php?id=7045

### CODE REVIEW

**Comment by ArneBab** [comment]
Did you see the comment about changes from the theme?

**Comment by ArneBab** [comment]
Arg, sorry, the comment wasn’t yet visible.

File: src/freenet/clients/http/WelcomeToadlet.java [hunk 1]
```diff
@@ -467,7 +467,7 @@ public void handleMethodGET(URI uri, HTTPRequest request, ToadletContext ctx) th
 			contentNode.addChild(ctx.getAlertManager().createSummary());
         }
 		
-        if (ctx.getPageMaker().getTheme().fetchKeyBoxAboveBookmarks) {
+        if (node.config.get("fproxy").getBoolean("fetchKeyBoxAboveBookmarks")) {
```
[the remark this answers is not in the document]
Comment (Olezha) [reply src/freenet/clients/http/WelcomeToadlet.java]: When the user changes the theme this setting takes the value specified in the new theme
https://github.com/freenet/fred/pull/655/commits/01ca314186512528d0ccc8abe0425aa66277a4a4#diff-f9b99757395e4e315c9cf7860995c0c7R263
But by the default this setting is `false` and probably also worth trying to capture the value from the theme.

[the remark this answers is not in the document]
Comment (ArneBab) [reply src/freenet/clients/http/WelcomeToadlet.java]: Can you still do the capturing?

**Comment by ArneBab** [comment]
looks good, thank you!

**Pull request merged**